layouts/partials/primary-download-matrix.hbs

```diff
@@ -64,12 +64,19 @@
       </tr>
 
       <tr>
-        <th>Linux Binaries (.tar.xz)</th>
+        <th>Linux Binaries (x86)</th>
```

Comment:
Not entirely sure about this, IMHO `x86` is the architecture platform, 32/64bit are different versions of it. Any thoughts @nodejs/website, @nodejs/build?